install_files/securedrop-app-code/debian/rules

```diff
@@ -2,7 +2,7 @@
 
 DEB_DH_INSTALL_ARGS=-X .git
 
-SECUREDROP_BUILD_PLATFORM=$(shell lsb_release -sc)
+SECUREDROP_BUILD_PLATFORM=$(shell awk -F= '$1=="VERSION_CODENAME" { print $2 ;}' /etc/os-release)
```

Comment:
Since this is Debian packaging, we already have access to the distro name. We have to add:

`include include /usr/share/dpkg/pkg-info.mk` to the top, which will create a `$DEB_DISTRIBUTION` variable for us.

Reply:
Actually, I grepped and don't see this used anywhere. We can just remove it entirely.

Reply:
Ok, going to remove it now.